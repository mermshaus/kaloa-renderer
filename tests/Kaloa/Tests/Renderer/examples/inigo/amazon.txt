[amazon asin=0201633612][/amazon]