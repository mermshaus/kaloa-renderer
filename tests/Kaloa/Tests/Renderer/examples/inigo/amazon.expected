<div class="amazon clear"><div class="img-border"><div class="img" style="background-image: url(http://images.amazon.com/images/P/sin=020163361.01._SCMZZZZZZZ_V65020934_.jpg);"></div></div><p class="title"><strong></strong></p><p class="author"></p></div>
